src/WebApp/RESTClients/IRepairManagementAPI.cs

```diff
@@ -14,11 +14,11 @@ public interface IRepairManagementAPI
 
     // Approve a repair order by ID
     [Post("/repairManagement/approve/{repairOrderId}")]
```

Comment:
dit zou ook niet werken want je stuut geen repairOrderId in de url tewijl je dit wel in de backend verwacht te krijgen

Reply:
Wat dan wel? Ik weet het niet echt.

Reply:
Task ApproveRepairOrder(string repairOrderId, [Body] ApproveRepairOrder command);